warehouse/models/intermediate/payments/int_payments__refunds_deduped.sql

```diff
@@ -0,0 +1,150 @@
+{{ config(materialized = "table") }}
+
+WITH
+
+micropayments_table_refunds AS (
+    SELECT
+
+        aggregation_id,
+        micropayment_id,
+        participant_id,
+        customer_id,
+        ABS(charge_amount) AS refund_amount,
+        EXTRACT(DATE FROM transaction_time AT TIME ZONE "America/Los_Angeles") AS transaction_date,
+
+        -- add columns that we want to preserve from refunds table after union as null strings
+        SAFE_CAST(NULL AS STRING) AS refund_id,
+        SAFE_CAST(NULL AS STRING) AS settlement_id,
+        SAFE_CAST(NULL AS STRING) AS retrieval_reference_number,
+        SAFE_CAST(NULL AS STRING) AS coalesced_id,
```

Comment:
For these refunds the aggregation ID should be used as the coalesced ID since they won't have an RRN